mastracode/factory-ui/src/ui/domains/factory/components/cardMorph.css

```
@@ -0,0 +1,94 @@
+/* The card's detail panel opens over the card itself, so it grows out of that
+ * top-left corner rather than fading in as a separate surface. Keyframes cannot
+ * be written as utilities, so the motion lives here; unlayered, so it beats the
+ * popover's stock zoom utilities. */
+
+.board-card-details[data-slot='popover-content'] {
+  --board-panel-w: min(28rem, calc(100vw - 1.5rem));
+  /* Replaced inline with the measured content height; this is only what the
+   * very first frame renders at, before the panel has been measured. */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)